index.html

```diff
@@ -2466,23 +2467,50 @@
                 Set <var>record</var>.data to a new <code>ArrayBuffer</code>
                 object constructed from the octets of <var>ndef.PAYLOAD</var>.
               </li>
-            </ul>
+            </ol>
           </li> <!-- parsing NDEF External/Unknown record -->
           <li>
+            Otherwise, skip to the next <a>NDEF record</a> in <var>input</var>.
+          </li>
+          <li>
             Add <var>record</var> to <var>message</var>.
+            Let the <var>receivedMessage</var> variable
+            <a href="#adapter-inbox">associated</a> with the current
+            <code><a>NFCAdapter</a></code> object be <var>message</var>.
           </li>
-        </ul>
+        </ol>
+      </li>
+      <li>
+        If <var>suspended</var> variable
+        <a href="#adapter-suspended">associated</a> with the current
+        <a>NFC adapter</a> is <code>false</code>, then for each
+        <a>NFC watch</a> <var>watch</var> registered using the
```

Comment:
`dispatch NFC content` also does its own iteration over NFC watches. You should only do it in one place.